Makefile

```diff
@@ -35,17 +36,27 @@ dist:
 		cd _dist && \
 		$(DIST_DIRS) cp ../LICENSE {} \; && \
 		$(DIST_DIRS) cp ../README.md {} \; && \
-		$(DIST_DIRS) tar -zcf $(SHORT_NAME)-${VERSION}-{}.tar.gz {} \; && \
-		$(DIST_DIRS) zip -r $(SHORT_NAME)-${VERSION}-{}.zip {} \; \
+		$(DIST_DIRS) tar -zcf $(SHORT_NAME)-${VERSION}-{}.tar.gz {} \; \
```

Comment:
MAY: use verbose for the tar creation to make the build log slightly more transparent.

Reply:
I've added the verbose flag.  I'm not sure why any of this is really needed because of goreleaser, but I suppose it doesn't hurt to have it.